resolwe_bio/processes/import_data/expressions.yml

```diff
@@ -59,6 +60,41 @@
           value: NCBI
         - label: UCSC
           value: UCSC
+    - name: species
+      label: Species
+      type: basic:string
+      description: >
+        Species latin name.
+      allow_custom_choice: true
+      choices:
+        - label: Homo sapiens
+          value: Homo sapiens
+        - label: Mus musculus
+          value: Mus musculus
+        - label: Rattus norvegicus
+          value: Rattus norvegicus
+        - label: Dictyostelium discoideum
+          value: Dictyostelium discoideum
+        - label: Odocoileus virginianus texanus
+          value: Odocoileus virginianus texanus
+        - label: Solanum tuberosum
+          value: Solanum tuberosum
+    - name: build
+      label: Build
+      type: basic:string
+      description: >
+        Genome build or annotation version.
+    - name: feature_type
+      label: Feature type
+      type: basic:string
+      default: gene
+      choices:
```

Comment:
@jkokosar I would suggest to add here: allow_custim_choice: true, if user has microRNA data, or some other not usual use cases.